ceph.spec.in

```diff
@@ -586,7 +586,6 @@ Summary:	Ceph daemon for immutable object cache
 Group:		System/Filesystems
 %endif
 Requires:	ceph-common = %{_epoch_prefix}%{version}-%{release}
-Requires:	libradospp1 = %{_epoch_prefix}%{version}-%{release}
```

Comment:
@tchaikov sorry for this mess.. 
The daemon needs to read rados objects from RADOS and store them on a local device.  if i read https://github.com/ceph/ceph/commit/65c8733b56d2392ec70d6bf4db994697a4828287 correctly, we still need this dependency here but need to change it to `librados2`?
`Requires:	librados2 = %{_epoch_prefix}%{version}-%{release}`

Reply:
s/ceph-common/librados2/

updated and repushed.